styleguide/styles.scss

```diff
@@ -62,6 +62,19 @@
   }
 }
 
+$cads-spacers: (
+  (
+    0: 0,
```

Comment:
`$cads-spacing-0` isn't used by any macros etc?

Reply:
It's never used. Plus having a variable name for 0 is a bit much. The Mayans already sorted this for us 🙃

Reply:
I figured it'd be handy in a loop, but if it's not used and brings us no joy. then 🧹. What year is the world meant to end again? ;)